index.html

```diff
@@ -6,7 +6,7 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Vite + React + TS</title>
   </head>
-  <body>
+  <body style="margin: 0">
```

Comment:
You can style the Appbar component to remove the margin - no need to mess with the html

Reply:
HTML edit removed, achieved by changing Navbar.tsx instead